include/ttool.hh

```diff
@@ -50,6 +50,7 @@ namespace ttool
         TTool(std::string ttoolRootPath, std::string configFile, std::string cameraCalibFile)
         {
             InitializeConfig(ttoolRootPath, configFile);
+            m_ConfigPtr->CheckAcitFiles(ttoolRootPath);
```

Comment:
As discussed, I think `m_ConfigPtr->CheckAcitFiles(ttoolRootPath);` should not be exposed in the constructor. 
I propose simply to put it in the definition of `InitializeConfig(..)` function, in the same `ttool.hh` header. Here:

https://github.com/ibois-epfl/TTool/blob/6e19a12bcba18326c3bfa8cd76d3e66bd2c27488/include/ttool.hh#L433C9-L438C10

```cpp
 void InitializeConfig(std::string ttoolRootPath, std::string configFile)
        {
            m_ConfigFile = configFile;
            m_ConfigPtr = std::make_shared<ttool::Config>(configFile);
            m_ConfigPtr->SetTToolRootPath(ttoolRootPath);

            # what I propose
            m_ConfigPtr->CheckAcitFiles(ttoolRootPath);
            m_ConfigPtr->CheckClassifierLabelsConfig();

        }
```
Let me know if it works, and give it a go on a compiled AC just to be 100% sure that there are not problems.